docs/_posts/2021-03-29-using-seahorn.md

```diff
@@ -0,0 +1,331 @@
+---
+layout: post
+title: "Using Seahorn"
+permalink: /using-seahorn/
+---
+
+(This post borrows heavily from the [Using KLEE] post)
+
+[SeaHorn] is an automated analysis framework for [LLVM]-based languages.
+The master branch of [SeaHorn] uses [LLVM] 5.0 at the moment, which is not
```

Comment:
In the first paragraph, focus on things like why SeaHorn is interesting and what this post is about.

Defer this bit about dev10 until later - maybe somewhere around line 60?
(And maybe as a footnote - depending on how important it is?)